lesson_04_homework.rb

```diff
@@ -1,50 +1,58 @@
 # Code Reading
 
-def fizzbuzz(num)
-  # Example comment line
-
-  case
-    when num % 15 == 0 then "FizzBuzz"
-    when num % 3  == 0 then "Fizz"
-    when num % 5  == 0 then "Buzz"
-    else num
-  end
-end
-
-def fizz_buzz_to(limit)
-  1.upto(limit).each do |num|
-    puts fizzbuzz(num)
-  end
-end
+# def fizzbuzz(num)
```

Comment:
🔨 @dnori817 - it looks like you commented everything instead of just adding comments within the working code.